aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/RawClientDecorator.kt

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+package software.amazon.smithy.rustsdk
+
+class RawClientDecorator
```

Comment:
What is this for?

Reply:
there's a refactoring in here to remove aws-hyper and instead have per-service defined middleware stacks (which is I think, what we want in the long term).

I think we can find a better name, but this is the "low level" client that exposes a `.call` method.

We could name it "tower service decorator" or something along those lines.